src/vs/workbench/contrib/tasks/browser/abstractTaskService.ts

```diff
@@ -2779,6 +2779,9 @@ export abstract class AbstractTaskService extends Disposable implements ITaskSer
 	}
 
 	private _runTaskCommand(arg?: any): void {
+		if (!this._tasksReconnected) {
```

Comment:
the tests are failing bc of this... not sure what to do about it

Reply:
oh I think I see the issue

Reply:
hmm seems that's not needed, but even when I hide the panel, it's getting shown on window reload